components/organisms/qrCode/qrcode.js

```diff
@@ -12,22 +12,22 @@ import QRCode from 'qrcode.react'
 // Style
 import styled from 'styled-components'
 
-const Qrcode = ({ assetId, includeMargin, level, path, renderAs, size }) => {
-  const downloadQR = () => {
-    console.log('HERE')
-    const canvas = document.querySelector(`#qr${assetId} > canvas`)
-    const pngUrl = canvas.toDataURL()
+export const downloadQR = (assetId) => {
```

Comment:
@Ojaghi1991 Only one export per file :)
Alwats default please to keep with V3 style